src/app/query-runner.service.ts

```diff
@@ -161,12 +160,26 @@ export class QueryRunnerService {
     query.duration = (new Date()).getTime() - query.requestSentAt.getTime();
     query.statusCode = res.status;
 
-    AppComponent.messageBarContent = {
+    AppComponent.requestStatus = {
       text: this.createTextSummary(query),
       backgroundClass: this.isSuccessful(query) ? 'ms-MessageBar--success' : 'ms-MessageBar--error',
       icon: this.isSuccessful(query) ? 'ms-Icon--Completed' : 'ms-Icon--ErrorBadge',
     };
 
+    const queries = [
+      'https://graph.microsoft.com/v1.0/me/',
+      'https://graph.microsoft.com/v1.0/user/',
```

Comment:
the endpoint is `users` not `user`

Not sure what the _match_ method does, but unclear if it will match
`../v1.0/users`
`../v1.0/users/{id}`

Also over time we might need to add more entity types in here.  One that comes to mind already is groups
../v1.0/groups
../v1.0/groups/{id}

So not sure if you `hasQuery` statement should use a loop around the `query` array to do the matching.

Reply:
This approach will not be able to match `../v1.0/users/{id}`. I am experimenting with regex to see how much flexibility it will give us.